### ISSUE TITLE
Testing with Azure on Windows

### ISSUE THREAD

**Issue by raholler** [hmgaudecker/econ-project-templates#66]
Implement at least subbattery of tests that do not rely on latex distribution.
Do some more research on whether there is any possibility to get latex in azure on windows.

**Comment by raholler** [comment]
https://github.com/QCoDeS/Qcodes/blob/master/azure-pipelines.yml

**Comment by hmgaudecker** [comment]
We should bump the minimum required conda version in the docs then, too. I  am at 4.7.12 now.

**Comment by hmgaudecker** [comment]
There is also recent [failures](https://dev.azure.com/hmgaudecker/econ-project-templates/_build/results?buildId=98) on the other platforms due to conda / some setuptools dependency:

```
RemoveError: 'setuptools' is a dependency of conda and cannot be removed from
conda's operating environment.
```

Quick googling suggests to me that this may be due to azure's conda being dated, so it may be worth just waiting for a little bit. No big cause of concern ATM.

**Comment by hmgaudecker** [comment]
Seems to be a more general issue https://github.com/conda/conda/issues/9337 - let's wait for a couple of days for what is happening there.

Although this approach https://github.com/conda/conda-prefix-replacement/blob/master/.travis.yml#L9-L14 is apparently more more robust on Travis -- would it also work on Azure? @tobiasraabe?

**Comment by tobiasraabe** [comment]
I used the hotfix explained in the same thread which uninstalls setuptools via pip first and then installs it again using conda. This is only necessary on Linux and MacOS. Windows seems fine.

Team Windows: 1
Team Rest: 0

:)

https://github.com/OpenSourceEconomics/respy/pull/260/files

**Comment by hmgaudecker** [comment]
Perfect, thanks! 

In the grand scheme of things in terms of CI, this seems closer to our experience, though :-) 

> Team Windows: 1
> Team Rest: ~0~ 5

**Comment by raholler** [comment]
error does not occur anymore, still action required?

**Comment by hmgaudecker** [comment]
The `RemoveError` -- probably no action required, seems like it was a general issue and hopefully they solved it on the conda (?) side, but I did not follow up on it.

**Comment by tobiasraabe** [comment]
We fixed the error in #70. I bookmarked the conda issue and report back if there is indeed a cleaner solution in the future.

**Comment by hmgaudecker** [comment]
Haha, thanks for the reminder, that's what happens my memory if I am doing 7 things at once...

**Comment by raholler** [comment]
Fixed in #74 but without Latex testing.
